src/js/nav-collapsible.js

```diff
@@ -0,0 +1,30 @@
+export function nav() {
+  "use strict";
+  // RESPONSIVE NAV JS
+  // http://jsfiddle.net/csswizardry/ev598/
+  // http://jsfiddle.net/shanomurphy/zp376/
+  // https://codepen.io/atelierbram/pen/ALzCs
```

Comment:
Source callouts like this are only necessary when doing something out of the ordinary.